packages/plugin-core/src/components/lookup/buttons.ts

```diff
@@ -424,6 +439,7 @@ export class HorizontalSplitBtn extends DendronBtn {
   static create(pressed?: boolean) {
     return new HorizontalSplitBtn({
       title: "Split Horizontal",
+      description: "Lookup result will be opened in a vertical split",
```

Comment:
I know this modifier naming is weird to begin with, but the title and description saying different things would make it even worse 😅  Maybe we should consolidate how we call these...

VSCode goes with "open X to the side". Maybe we should follow that as well?

Reply:
Sounds good. I can change the title to make the description. Can you clarify what you mean by "open X to the side"?

Reply:
Just synced with Kevin and we're good with changing the description to follow VSCode. I'll change the title later in a follow up commit since it'll be breaking